test/lib/independence/index.test.js

```diff
@@ -0,0 +1,105 @@
+const chai = require('chai')
+const sinon = require('sinon')
+const spies = require('chai-spies')
+const axios = require('axios')
+const proxyquire = require('proxyquire')
+const Promise = require('bluebird')
+
+chai.use(spies)
+
+const requestMock = {
+  send: () => Promise.resolve(mockResponse)
+}
+const mockResponse = {
+  data: { field1: 'value1' },
+  status: 200
+}
+let mockAdapter
+let pendingRequests
+const IndependentRequest = chai.spy(data => {
+  requestMock.data = data
+  return requestMock
+})
+
+const { makeIndependent } = proxyquire('../../../lib/independence', {
+  './independent_request': {
+    IndependentRequest
+  }
+})
+
+const expect = chai.expect
+
+function retryedPendingRequests () {
+  expect(IndependentRequest.all).to.have.been.called()
+  pendingRequests.forEach(request => {
+    expect(request.send).to.have.been.called()
+  })
+}
+
+function didNotRetryPendingRequests () {
+  expect(IndependentRequest.all).to.not.have.been.called()
+  pendingRequests.forEach(request => {
+    expect(request.send).to.not.have.been.called()
+  })
+}
+
+describe('independence', () => {
+  let client
+  let clock
+
+  beforeEach(() => {
+    mockAdapter = () => Promise.resolve(mockResponse)
+    pendingRequests = [{
```

Comment:
either start each object on it's own line or write each object in one line (which then you will need to start it on it's own line anyway)